src/QueueListNPC.cpp

```diff
@@ -0,0 +1,45 @@
+#include "QueueListNPC.h"
+#include "QueueListCache.h"
+
+enum Gossips
+{
+    Show_RatedArena_Queues = 1,
+    Show_SkirmishArena_Queues = 2,
+    Show_Battleground_Queues = 3
+};
```

Comment:
you could move this to the QueueListNPC.h file, moreover the constants variables usually are written in UPPER_CASE
```
Identifiers representing [macros](https://en.wikipedia.org/wiki/C_preprocessor#Macro_definition_and_expansion) are, by convention, written using only uppercase letters and underscores (this is related to the convention in many programming languages of using all-upper-case identifiers for constants).
```
source https://en.wikipedia.org/wiki/Naming_convention_(programming) section C and C++